features/cart/shopping_cart/viewing_cart_summary.feature

```diff
@@ -11,3 +11,12 @@ Feature: Viewing a cart summary
     Scenario: Viewing information about empty cart
         When I see the summary of my cart
         Then my cart should be empty
+
+    @ui @no-api
+    Scenario: Viewing information about empty cart after clearing cookies
+        Given the store has a product "T-Shirt banana" priced at "$12.54"
+        And I added this product to the cart
+        And I am on the summary of my cart page
+        And my session expired
```

Comment:
`my session expired` is quite a technical detail :dancer: what is the observable behaviour the Customer can experience? Probably something like `But then I am gone for a long time` 😄

Reply:
Indeed it's a technical detail, but at the same time the bug is also strictly technical issue 🤷🏼‍♂️. I'll change it.